dts/arm/sam_e70_xplained.fixup

```diff
@@ -5,3 +5,18 @@
  */
 
 #define CONFIG_NUM_IRQ_PRIO_BITS	ARM_V7M_NVIC_E000E100_ARM_NUM_IRQ_PRIORITY_BITS
+
+#define CONFIG_UART_SAM_PORT_0_NAME	ATMEL_SAM_UART_400E0800_LABEL
+#define CONFIG_UART_SAM_PORT_0_BAUD_RATE	ATMEL_SAM_UART_400E0800_CURRENT_SPEED
+#define CONFIG_UART_SAM_PORT_1_NAME	ATMEL_SAM_UART_400E0A00_LABEL
+#define CONFIG_UART_SAM_PORT_1_BAUD_RATE	ATMEL_SAM_UART_400E0A00_CURRENT_SPEED
+#define CONFIG_UART_SAM_PORT_2_NAME	ATMEL_SAM_UART_400E1A00 _LABEL
+#define CONFIG_UART_SAM_PORT_2_BAUD_RATE	ATMEL_SAM_UART_400E1A00_CURRENT_SPEED
+#define CONFIG_UART_SAM_PORT_3_NAME	ATMEL_SAM_UART_400E1C00_LABEL
+#define CONFIG_UART_SAM_PORT_3_BAUD_RATE	ATMEL_SAM_UART_400E1C00_CURRENT_SPEED
```

Comment:
Sorry for being late. This PR looks very good but there is one omission, the #defines for CONFIG_UART_SAM_PORT_4_NAME, CONFIG_UART_SAM_PORT_4_BAUD_RATE are missing. @fallrisk could you prepare another PR?